packages/xstate-inspect/rollup.config.js

```diff
@@ -30,6 +34,34 @@ const createNpmConfig = ({ input, output }) => ({
   plugins: [createTsPlugin()]
 });
 
+function createUmdConfig({ input, output: file, name }) {
+  return {
+    input,
+    external: makeExternalPredicate(Object.keys(pkg.peerDependencies)),
+    output: {
+      file,
+      format: 'umd',
+      name,
+      globals: {
+        react: 'React',
+        xstate: 'XState',
+        '@xstate/fsm': 'XStateFSM'
```

Comment:
I don't think that `@xstate/inspect` is using `@xstate/fsm` anyhow - not the latter is compatible with the inspect package
```suggestion
```

Reply:
ok, fixed.